packages/valory/skills/apy_estimation_abci/tests/test_behaviours.py

```diff
@@ -204,6 +204,7 @@ def setup_class(cls, **kwargs: Any) -> None:
     def setup(self, **kwargs: Any) -> None:
         """Set up the test method."""
         super().setup()
+        assert self.behaviour.current_behaviour is not None
```

Comment:
Added this check for linters: `self.behaviour.current_behaviour` is optional, so `None` does not have `params`. We exclude that possibility with the assertion and the linters pass.